frontend/src/components/Layouts/Header/Header.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import BurgerMenu from './Components/BurgerMenu/BurgerMenu';
+import styles from './Header.module.scss';
+import Button from '../../ui-kit/components/Button';
+import { ICONS } from '../../ui-kit/icons';
+import InputSearch from '../../ui-kit/components/Input/InputSearch';
+import ROUTES from '../../../constants/routers';
+import IconButton from '../../ui-kit/components/IconButton';
+import { navigationItems } from './navigationItems';
+import NavigationMenu from './Components/NavigationMenu/NavigationMenu';
+
+const Header = () => {
+  const [sticky, setSticky] = useState(false);
+
+  useEffect(() => {
```

Comment:
Without an array of dependencies, useEffect makes no sense, it should not be triggered for every user action